### PR TITLE
Fix selected tab on multi-language 

### LINKED ISSUE #4071: Selected site tab resets after language change
## Description
After changing the language with a selected site tab, you will be redirected to the first tab. Sometimes the translated content was not saved in the backend (there is always the value from the default language) even though it was changed correctly in the content text file.

## To reproduce

1. Install a fresh Starterkit and create tabs in the site.yml file
2. Add multilanguage support to the kirby installation
3. Add a field to the tabs
4. Click on a tab and switch language
5. You will be redirected to the first tab

-- For the problem that some values are not shown in the right language in the backend i could provide you my kirby project.

## Your setup
3.6.1.1
 - Device: PC
 - OS: Windows
 - Browser: Google Chrome
 - Version: 96.0.4664.110

### PR DESCRIPTION
**Pull request by afbora**
## Describe the PR

Since tabs work with query string, an additional `tab` parameter has been added to the language switcher.

If you have a more efficient or simpler method than the one I use (URLSearchParams), I'd appreciate it.

### Fixes

- Fixed selected tab on multi-language #4071

## Breaking changes

None

## Related issues/ideas

- Fixes #4071

## Ready?

- [ ] Unit tests for fixed bug/feature
- [x] In-code documentation (wherever needed)
- [x] CI checks pass

## When merging

- [x] Add to [website docs release checklist](https://github.com/getkirby/getkirby.com/pulls) (if needed)
- [x] Add changes to release notes draft in Notion

### CODE REVIEW

**Comment by afbora** [comment]
~I don't know why github actions timeout and canceled.. 🤷‍♂️~

There was an incident with GitHub Actions and fixed. Now all test passed!

File: panel/src/components/Navigation/Languages.vue [hunk 1]
```diff
@@ -38,11 +38,18 @@ export default {
...
+        language: language.code
+      };
+
+      // append if any tab is active
+      const params = new URLSearchParams(window.location.search);
+      if (params.has("tab")) {
+        query.tab = params.get("tab");
+      }
+
+      this.$go(this.$view.path, { query: query });
```
Comment (johannschopplich) [inline panel/src/components/Navigation/Languages.vue]: `{ query: query }` could be simplified to just `{ query }`. 🙂

Comment (afbora) [reply panel/src/components/Navigation/Languages.vue]: Thanks for pair of your eyes 👍

**Review by bastianallgeier — APPROVED** [review]
I simplified it by passing `window.location`. The underlying URL builder will take the Location object and inject the query parameters into the existing query. This will only overwrite the specified parameters and leave the rest as they are.

**Comment by afbora** [comment]
Thanks for a more comprehensive solution 👍

**Pull request merged**